cmds/exp/tcpdump/gen.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build ignore
+
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	file, err := os.Create("well_known_ports.go")
+	if err != nil {
+		panic(err)
```

Comment:
This is one case where you should never, ever use panic. log.Fatalf, sure, but panic? no.

Reply:
Fixed